hack/update-deepcopy.sh

```diff
@@ -7,7 +7,7 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-ge
 
 verify="${VERIFY:-}"
 
-${CODEGEN_PKG}/generate-groups.sh "deepcopy" \
+bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy" \
```

Comment:
what does it help with if it already has `#!/bin/bash`

Reply:
`go mod vendor` drops executable bit from files.

Reply:
known workaround.